src/adapter.coffee

```diff
@@ -77,7 +77,7 @@ Rivets.public.adapters['.'] =
                 callbacks = map.callbacks
 
                 if callbacks[keypath]
-                  callback() for callback in callbacks[keypath].slice() when callback in callbacks[keypath]
```

Comment:
@blikblum I understand this now. Why do you think the original author was cloning the array and checking with `when`? Is there a reason you haven't followed this?